src/student_success_tool/modeling/evaluation.py

```diff
@@ -644,6 +644,59 @@ def plot_shap_beeswarm(shap_values):
     return plt.gcf()
 
 
+def fnpr_group_plot(fnpr_data: list) -> matplotlib.figure.Figure:
+    """
+    Plots False Negative Prediction Rate (FNPR) for a group by subgroup on
+    a split (train/test/val) of data with confidence intervals.
+
+    Parameters:
+    - fnpr_data: List with dictionaries for each subgroup. Each dictionary
+    comprises of group, fnpr, ci (confidence interval), and split_name keys.
```

Comment:
google-style docstring, please!

```suggestion
    Args:
        fnpr_data: List with dictionaries for each subgroup. Each dictionary
            comprises of group, fnpr, ci (confidence interval), and split_name keys.
```